packages/ui/package.json

```diff
@@ -43,8 +43,8 @@
   },
   "homepage": "https://github.com/draftbit/react-native-jigsaw#readme",
   "dependencies": {
-    "@draftbit/core": "^43.5.0",
-    "@draftbit/native": "^43.4.19"
+    "@draftbit/core": "^43.6.0-alpha.6",
```

Comment:
is this right? maps seems to be 43.5.0?

Reply:
That got pushed during my testing, just reverted